packages/css/types/index.d.ts

```diff
@@ -14,6 +18,6 @@ export { Interpolation, SerializedStyles }
 
 export default function css(
   template: TemplateStringsArray,
-  ...args: Array<Interpolation>
+  ...args: Array<CSSInterpolation>
 ): SerializedStyles
-export default function css(...args: Array<Interpolation>): SerializedStyles
+export default function css(...args: Array<CSSInterpolation>): SerializedStyles
```

Comment:
Change log:

`css` function has been restricted to prevent passing of invalid types